std/meta/list.d

```diff
@@ -0,0 +1,1038 @@
+// Written in the D programming language.
+
+/**
+ * Templates with which to manipulate
+ * $(LINK2 ../template.html#TemplateArgumentList, $(I TemplateArgumentList))s.
+ * Such lists are known as type lists when they only contain types.
+ *
+ * Some operations on template argument lists are built in to the language,
+ * such as $(D Args[$(I n)]) which gets the $(I n)th element from the
+ * _list. $(D Args[$(I lwr) .. $(I upr)]) returns a new
+ * _list that is a slice of the old one.
```

Comment:
Nitpick: Could be worth to note their similarity with arrays here. Everyone knows how to work with arrays.